v2/devices/miatoll.yml

```diff
@@ -29,6 +29,14 @@ user_actions:
     title: "OEM unlock"
     description: "If you haven't done so already, make sure to OEM unlock your device first."
     link: "https://en.miui.com/unlock/"
+  datawarning:
+    title: "All your userdata will be lost"
+    description: "Flashing Droidian will wipe all the data saved on the device including your personal files. Please backup everything before continuing!"
+    button: true
+  droidian_alpha:
+    title: "Alpha quality software"
+    description: "Remember that Droidian is alpha quality software, absolutely not daily driver ready. If you break your device, you get to keep both pieces!"
```

Comment:
I feel like this warning discredits Droidian.

If the system is that unstable to warrant such a warning, i would prefer to not submit this PR yet.

Reply:
Agreed, please remove that.

Reply:
Ah it's just the warning? I guess I could remove it :) I've took an example from another config here which has the same warning.

Reply:
now i see https://github.com/ubports/installer-configs/pull/102/commits/04f77c0d971cd6bc55e74a344ad152567ab7412f